assets/js/pages/AnalyticsEula/AnalyticsEula.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { setUser } from '@state/user';
+
+import { isAdmin } from '@lib/model/users';
+import { editUserProfile } from '@lib/api/users';
+import { getFromConfig } from '@lib/config/config';
+import { getUserProfile } from '@state/selectors/user';
+import AnalyticsEulaModal from './AnalyticsEulaModal';
+
+const analyticsEnabledConfig = getFromConfig('analyticsEnabled');
+
+export default function AnalyticsEula() {
+  if (!analyticsEnabledConfig) {
+    return null;
+  }
+
+  const { analytics_eula_accepted, username } = useSelector(getUserProfile);
```

Comment:
nitpick: 
maybe this looks more understandable, instead of creating a user const below:
```
const user = useSelector(getUserProfile);
const { analytics_eula_accepted, username } = user;
...
  const isDefaultAdmin = isAdmin(user);
```
What do you think?

Reply:
Cool, I have made this change. I think it reduces some confusion in the code.